src/text_shaper/directwrite_shaper.cpp

```diff
@@ -66,6 +66,60 @@ namespace
         }
         return font_slant::normal;
     }
+
+    void renderGlyphRunToBitmap(IDWriteGlyphRunAnalysis* _glyphAnalysis, RECT& _textureBounds, bitmap_format _targetFormat, std::vector<uint8_t>::iterator& _it)
```

Comment:
I am on smartphone, how do I trigger a code change propose when not being able to press Ctrl+g? Hmm...

The parameter `_textureBounds` can be passed as const reference here.

As of https://docs.microsoft.com/en-us/windows/win32/api/dwrite/nf-dwrite-idwriteglyphrunanalysis-createalphatexture

Reply:
Ah thanks. It should.